core/tests/protractor_utils/general.js

```diff
@@ -128,16 +134,25 @@ var moveToEditor = async function() {
 };
 
 var expect404Error = async function() {
-  expect(await element(by.css('.protractor-test-error-container')).getText()).
+  var errorContainer =
+    element(by.css('.protractor-test-error-container'));
```

Comment:
For statements spanning multiple lines, wrap inside `(..)` and always break after `(`.

See [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript):

>We use extra parentheses if a statement breaks across multiple lines, similar to Python. In particular, when code in '(...)' or '[...]' spans more than one line, make a line break after the opening parentheses or bracket.

Reply:
I have made the I will take care next time onward. Thanks!